### PR TITLE
Implement balances on currency selection

### PR DESCRIPTION
**Pull request by jamesondh**
- Reading NEAR account balance works
- Tweak input card and currency selection design
- Rename global context to inputs context to reflect its scope more accurately
- Add tags for each currency in token list (ERC-20, NEP-21, Native token (NEAR), etc.)
- Loading ERC-20 balances does not yet work (see /services/web3utils)

### CODE REVIEW

**Review by robert-zaremba — APPROVED** [review]

**Comment by robert-zaremba** [comment]
@jamesondh  - you can create branches directly in this repository. It makes easier to do merges etc...

**Comment by robert-zaremba** [comment]
@amityadav0  - please also do a full review here.

File: webapp/package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "react": ">=16.3.0",
     "react-bootstrap": "^1.3.0",
     "react-dom": "^16.13.1",
+    "react-hook-thunk-reducer": "^0.2.4",
     "react-icons": "^3.11.0",
     "react-router-dom": "^5.2.0",
     "regenerator-runtime": "^0.13.5"
```
Comment (amityadav0) [inline webapp/package.json:43]: require('string_decoder') this is missing from package.json, i encountered a build error

File: webapp/src/services/web3utils.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  new Promise((resolve, reject) =>
+    inner((err, res) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(res);
+      }
+    })
+  );
+
```
Comment (amityadav0) [inline webapp/src/services/web3utils.js:13]: I think we won't need this function,
we can use await directly in getBalance

Comment (jamesondh) [reply webapp/src/services/web3utils.js]: Removing that and getBalance since I don't think we need ETH balance for now

**Review by amityadav0 — APPROVED** [review]

**Pull request closed without merging**